environments/test.env

```diff
@@ -0,0 +1,26 @@
+NODE_ENV=test
+HTTP_PROXY=
```

Comment:
This should be enough to set it here instead of whole command.
It should be set in test setup (so before any of our code runs in tests)